unl_fourone.info.yml

```diff
@@ -5,6 +5,13 @@ core: 8.x
 
 libraries:
   - unl_fourone/global-styling
+  - unl_fourone/idm
+  - unl_fourone/amd
+
+libraries-override:
+  core/classList: false
```

Comment:
classList specifically because it is _only_ for IE <= 9, which are out of our support target (removing unnecessary cruft).